bin/run_tests

```diff
@@ -9,8 +9,8 @@ do
   echo "Testing: $test"
   total=$(( $total + 1 ))
   # Result will tell us how many lines are different between the expected dot file and the generated one.
-  result=`(cd $test ; viz_pipe config* ; (diff -y --suppress-common-lines expected.dot config*.dot | wc -l))`
-  if [[ $result -ne 0 ]]
+  result=`(cd $test ; viz_pipe config* ; cmp --silent expected.dot config*.dot) || echo 'fail'`
+  if [[ "$result" =~ "fail" ]]
```

Comment:
![image](https://user-images.githubusercontent.com/67082626/95515101-14dc0600-098b-11eb-95c0-219a27be3a5a.png)
I'm not exactly sure what's going on here but all the test failed for me,

Reply:
I'll look into this now.

Reply:
Ah I found whats wrong. Seems like the indexing of the associative arrays differs between machines.